d2l-table-style.html

```diff
@@ -0,0 +1,309 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../d2l-colors/d2l-colors.html">
+
+<!--
+This module styles table elements under <content> tags.
+
+	<style include="d2l-table-style"></style>
+	<table class="d2l-table">
+		<template>
+			<content></content>
+		</template>
+	</table>
+	<div class="d2l-table"><content></content></div>
+
+TODO: When ShadowDOM v1 is available, use <slot> and ::slotted
+-->
+<dom-module id="d2l-table-style">
+	<template>
+		<style include="d2l-colors">
+			.d2l-table {
```

Comment:
That is correct.

In order to get a scrollbar, we need a wrapper element. So, I'm setting the display of the table to block, and the display of the div to table, which I am aware is super hacky.

I'm not sure of the best way to go about it. In order to use the current table elements, the elements must be within a table element, hence the need to extend it. I can't create a `<d2l-table>` and expect it to work.

An alternative to this "inner" div approach is to create substitutes of the table elements, so we can bypass the HTML limitations